src/pages/indicatorTypes/form/IndicatorTypesFormFields.tsx

```diff
@@ -0,0 +1,48 @@
+import i18n from '@dhis2/d2-i18n'
+import { InputFieldFF } from '@dhis2/ui'
+import React from 'react'
+import { Field } from 'react-final-form'
+import {
+    CustomAttributesSection,
+    StandardFormField,
+    StandardFormSection,
+    StandardFormSectionTitle,
+    StandardFormSectionDescription,
+} from '../../../components'
+
+export const IndicatorTypesFormFields = () => {
+    return (
+        <>
+            <StandardFormSection>
+                <StandardFormSectionTitle>
+                    {i18n.t('Basic information')}
+                </StandardFormSectionTitle>
+                <StandardFormSectionDescription>
+                    {i18n.t(
+                        'Set up the basic information for this Indicator Type.'
+                    )}
+                </StandardFormSectionDescription>
+
+                <StandardFormField>
+                    <Field
```

Comment:
You could also use the re-usable `NameField` component - which has some additional validations, like checking if the value is taken.

Reply:
I am adding this to the next commit.